Metalama.Framework.DesignTime/Preview/ITransformationPreviewServiceImpl.cs

```diff
@@ -10,5 +10,6 @@ public interface ITransformationPreviewServiceImpl : IGlobalService
     Task<SerializablePreviewTransformationResult> PreviewTransformationAsync(
         ProjectKey projectKey,
         string syntaxTreeName,
+        IEnumerable<string> additionalSyntaxTreeNames,
```

Comment:
It's not clear why `syntaxTreeName` and `additionalSyntaxTreeNames` could not be a single parameter `syntaxTreeNames`. Document or merge.

Reply:
Please document why you have two parameters instead of one.